ecosystem/sep-0031.md

```diff
@@ -504,12 +510,12 @@ Name | Type | Description
 `status` | string | The status of the transaction. Values are outlined below.
 `status_eta` | number | (optional) The estimated number of seconds until a status change is expected.
-`amount_in` | string | (optional) The amount of the Stellar asset received or to be received by the Receiving Anchor. Excludes any fees charged after Receiving Anchor receives the funds.
+`amount_in` | string | (optional) The amount of the Stellar asset received or to be received by the Receiving Anchor. It should be equals to the `amount` value used in the [`POST /transactions`](#post-transactions) request. If a `quote_id` was provided in the [`POST /transactions`](#post-transactions) request, the quote's `sell_amount` should also be equals to `amount_in`. </br></br>The Receiving Anchor may extract fees from this amount before converting it to the amount_out.
 `amount_in_asset` | string | (optional) The asset received or to be received by the Receiving Anchor. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_out` | string | (optional) The amount sent by the Receiving Anchor to Receiving Client.
-`amount_out_asset` | string | (optional) The asset delivered to the Receiving Client. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
-`amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor.
-`amount_fee_asset` | string | (optional) The asset in which fees are calculated in. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_out` | string | (optional) The amount sent or to be sent by the Receiving Anchor to the Receiving Client. When using a `quote_id`, `amount_out` should always be the same as the quote's `buy_amount`. When using a `destination_asset` in the [`POST /transactions`](#post-transactions) request, it's expected that this value is only populated after the Receiving Anchor receives the incoming payment.
+`amount_out_asset` | string | (optional) The asset delivered to the Receiving Client. Must be present if `quote_id` or `destination_asset` was included in the `POST /transactions` request. If `destination_asset` was used when creating the transaction, `amount_out_asset` should be equals to `destination_asset`. </br></br>The value must be in [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format).
+`amount_fee` | string | (optional) The amount of fee charged by the Receiving Anchor. If a `quote_id` was used, fees should be detailed in the quote's `price_details` field.
```

Comment:
If a quote_id was used, `amount_fee` field should be left empty and detailed in the quote's price_details field.